seed/tests/test_analysis_pipelines.py

```diff
@@ -1076,7 +1076,7 @@ def test_get_location(self):
         location, status = _get_location(self.property_view)
         self.assertEqual(status, "success")
         self.assertTrue(location is not None)
-        self.assertEqual(location, "730 Garcia Street, Boring, Oregon, 97080")
+        self.assertEqual(location, "1070 Hunt Street, Boring, Oregon, 97015")
```

Comment:
just curious, why?

Reply:
Pretty sure this one comes from bumping Faker from `v0.9.3` to `v29.0.0`. Somewhere along the way they changed their algorithm for generating addresses and such